src/components/welcome-screen/welcome-screen.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const WelcomeApp = (props) => {
+  // eslint-disable-next-line react/prop-types
+  const {errorCount} = props;
+
+  return <section className="welcome">
+    <div className="welcome__logo"><img src="img/melody-logo.png" alt="Угадай мелодию" width="186" height="83" /></div>
+    <button className="welcome__button"><span className="visually-hidden">Начать игру</span></button>
+    <h2 className="welcome__rules-title">Правила игры</h2>
+    <p className="welcome__text">Правила просты:</p>
+    <ul className="welcome__rules-list">
+      <li>Нужно ответить на все вопросы.</li>
+      <li>Можно допустить {errorCount} ошибки.</li>
+    </ul>
+    <p className="welcome__text">Удачи!</p>
+  </section>;
+};
```

Comment:
Если верстку делать с одинаковыми отступами для открывающих и закрывающих скобок, читать потом легче :)
```suggestion
  return (
  <section className="welcome">
    <div className="welcome__logo"><img src="img/melody-logo.png" alt="Угадай мелодию" width="186" height="83" /></div>
    <button className="welcome__button"><span className="visually-hidden">Начать игру</span></button>
    <h2 className="welcome__rules-title">Правила игры</h2>
    <p className="welcome__text">Правила просты:</p>
    <ul className="welcome__rules-list">
      <li>Нужно ответить на все вопросы.</li>
      <li>Можно допустить {errorCount} ошибки.</li>
    </ul>
    <p className="welcome__text">Удачи!</p>
  </section>);
};
```